.github/dependabot.yml

```diff
@@ -0,0 +1,112 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
How does this step differ from the rest? Why "github-actions"?

Reply:
Github-actions are actions that are used in the workflows defined in .github/workflows/*.yml 
Thoses actions are versioned (v1.0.0, for example ). This élément of configuration is hère to check if there are news versions available for thé actions used in your workflow.

Thé other are configurijg thé check for python dependencies withe thé path their respective "dependency manager".